Homework15/component/src/avatar.js

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+
+const Avatar = (props) => {
+    return (
+        <div className="Emperor">
+            <div className="Info">
+                <img className="picture" src={props.author.pictureUrl} />
```

Comment:
Please also add an alt prop to the image. React gives a warning because of this
![Screenshot 2020-10-11 at 11 28 03](https://user-images.githubusercontent.com/11528060/95673824-d4f86700-0bb4-11eb-8d13-a6d3894959e1.png)